test/hack/e2e_scripts/install_karpenter.sh

```diff
@@ -1,7 +1,12 @@
 aws eks update-kubeconfig --name "$CLUSTER_NAME"
 
-# Parse minor version to determine whether to enable the webhooks
-K8S_VERSION_MINOR="${K8S_VERSION#*.}"
+# First, conditionally install the webhook stanza and CRDs
+if (( "$WEBHOOKS_ENABLED" == 'false' )); then
+helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd \
+  --namespace kube-system \
+  --version "0-$(git rev-parse HEAD)" \
+  --set webhook.enabled=${WEBHOOKS_ENABLED}
```

Comment:
Should we add `--wait`?

Reply:
What do we get with it? I also need to change the version, that's not correct as written.